src/components/ActionButton/ActionButton.vue

```diff
@@ -44,7 +44,9 @@ If you're using a long text you can specify a title
 
 <template>
 	<li :class="{ 'action--disabled': disabled }">
-		<button class="action-button"
+		<button
+			class="action-button"
+			:aria-label="ariaLabel"
```

Comment:
I guess this could even get to the ActionTextMixin ?
And be allowed in all ActionButton/Link/Router?

Reply:
You mean the prop??

Reply:
prop yes :)
But also to the other templates then :thinking:

Reply:
Only link and router?

Reply:
Well, All the ones that are allowed as single action, so ActionLink, ActionRouter & ActionButton
https://github.com/nextcloud/nextcloud-vue/blob/0f1a4f317763efeb8beb0cf742245c554b731f6a/src/components/Actions/Actions.vue#L249